src/components/DatetimePicker/DatetimePicker.vue

```diff
@@ -82,6 +82,7 @@ export default {
 		:type="type"
 		:value="value"
 		:append-to-body="false"
+		:input-attr="inputAttr"
 		v-bind="$attrs"
```

Comment:
We already bind all props here, I'm confused to why it doesn't work.

Reply:
Ah, because it needs an object.
Yeah, then let's use input-attr in forms directly then, this is why we added `v-bind="$attrs"`

Reply:
As the vue2-datepicker does not take `required` as prop. So this is why the way through `inputAttr` is necessary. I could do that also directly in forms, i just thought it is a bit nicer to have the required-prop here directly accessible.

Also see https://github.com/mengxiong10/vue2-datepicker/blob/2eea21fe450df981ae45bd18e69394af4244619a/src/date-picker.js#L446

Reply:
:man_shrugging:

Reply:
> i just thought it is a bit nicer to have the required-prop here directly accessible

Yeah it's a fair point. While we should stay as close as the original lib when wrapping components like that (datepicker, multiselect...) I'm also conflicted by staying as close as the html standards :see_no_evil: 

Both are valid points I think. @ChristophWurst @juliushaertl any thoughts?

Reply:
In any case, by doing this you remove the inputAttr from the available properties of the datetimepicker, which we should not do.
If we'd implement the required prop here, we should still keep both I think?

Reply:
Any opinion, @ChristophWurst @juliushaertl ?
So should i reopen and fix this PR?